### PR TITLE
Support release images for Kwok

### PR DESCRIPTION
**Pull request by wzshiming**
xref https://github.com/kubernetes/test-infra/pull/27128

### CODE REVIEW

File: groups/sig-scheduling/groups.yaml [hunk 1]
```diff
@@ -81,6 +81,16 @@ groups:
...
 
+  - email-id: k8s-infra-staging-kwok@kubernetes.io
+    name: k8s-infra-staging-kwok
+    description: |-
+      ACL for staging kwok
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - hweicdl@gmail.com
+      - zjwzshiming@gmail.com
```
Comment (wzshiming) [inline groups/sig-scheduling/groups.yaml:92]: @yibozhuang fill your email?

Comment (yibozhuang) [reply groups/sig-scheduling/groups.yaml:92]: thanks @wzshiming 
its: `yibzhuang@gmail.com`

File: k8s.gcr.io/manifests/k8s-staging-kwok/promoter-manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+registries:
+- name: gcr.io/k8s-staging-kwok
+  src: true
+- name: us.gcr.io/k8s-artifacts-prod/kwok
+  service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
+- name: eu.gcr.io/k8s-artifacts-prod/kwok
+  service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
+- name: asia.gcr.io/k8s-artifacts-prod/kwok
+  service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
+imagesPath: "../../images/k8s-staging-kwok/images.yaml"
```
Comment (ameukam) [inline k8s.gcr.io/manifests/k8s-staging-kwok/promoter-manifest.yaml:11]:
```suggestion
```
There is no need to do this. the promoter will look for this path.

**Comment by ameukam** [comment]
Project created:

```bash
Configuring special cases for Release Managers 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-artifact-promoter 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-build-image 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-ci-images 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-cip-test 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-experimental 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-kubernetes 
  Empowering kubernetes-release-test GCB service account to admin GCR 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-releng 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-releng-test 
  Empowering k8s-infra-release-viewers@kubernetes.io as project viewers in k8s-staging-publishing-bot 
Done 
```

**Pull request merged**